File: Code 3 Callouts/Code 3 Callouts/Models/Interiors/Residence.vb
Imports Rage

Namespace Models.Interiors

    Public Class Residence
        Public Sub New()
            Init()
        End Sub

        Public Sub New(ByVal pInterior As Interior, ByVal pEntryPoint As Vector3)
            Interior = pInterior
            EntryPoint = pEntryPoint
        End Sub

        Private Sub Init()
            Interior = Nothing
            EntryPoint = Vector3.Zero
        End Sub

        Public Property Interior As Interior
        Public Property EntryPoint As Vector3
    End Class

End Namespace

File: Code 3 Callouts/Code 3 Callouts/Models/Interiors/Residence.cs
using Rage;

namespace Stealth.Plugins.Code3Callouts.Models.Interiors
{

	internal class Residence
	{
		internal Residence()
		{
			init();
		}

		internal Residence(Interior pInterior, Vector3 pEntryPoint)
		{
			Interior = pInterior;
			EntryPoint = pEntryPoint;
		}

		private void init()
		{
			Interior = null;
			EntryPoint = Vector3.Zero;
		}

		internal Interior Interior { get; set; }
		internal Vector3 EntryPoint { get; set; }
	}

}